vendor/assets/stylesheets/dvl/components/sidebar_sub_actions.scss

```diff
@@ -5,5 +5,8 @@
   font-size: $fontSmall;
   .sidebar_box + & {
```

Comment:
I know this isn't part of your commit, but I think this is a hack.... if we want to conditionally display the sidebar sub actions on the right vs. the left, we should do that with a modifier, e.g. `sidebar_sub_actions.is_right`, yeah?